### PR TITLE
Add client id header for Swaps Apis

### PR DESCRIPTION
**Pull request by meppsilon**
Explanation: 
For the swaps APIs, we want to see the client id in the header of each request. It gives us more visibility into the client for each request and as a result, we can have better monitoring.
This PR sends the extension client id in the request header for requests sent to the Swaps APIs.

Manual testing steps:  
  - 
  - 
  -

### CODE REVIEW

**Comment by meppsilon** [comment]
I have read the CLA Document and I hereby sign the CLA

**Review by dan437 — COMMENTED** [review]
Thanks a lot for adding it, it looks great! There are gas-api calls outside of swaps in the extension (GasFeeController), so we might want to add it there as well: https://github.com/MetaMask/metamask-extension/blob/develop/app/scripts/metamask-controller.js#L216-L217

GasFeeController is in the `@metamask/controllers` node module.

File: ui/pages/swaps/swaps.util.js [hunk 1]
```diff
@@ -464,10 +466,11 @@ export async function fetchTokenBalance(address, userAddress) {
 }
 
 export async function fetchSwapsGasPrices(chainId, useNewSwapsApi) {
+  console.log('fetch gas prices');
```
[the remark this answers is not in the document]
Comment (meppsilon) [reply ui/pages/swaps/swaps.util.js]: Thanks for noticing this

**Review by dan437 — APPROVED** [review]
Looks great!

**Review by David2778 — APPROVED** [review]

**Review by darkwing — APPROVED** [review]

**Pull request merged**